build.gradle.kts

```diff
@@ -28,6 +28,7 @@ val e2eTest =
         group = "verification"
         filter {
             includeTestsMatching("*.e2e.*")
+            excludeTestsMatching("com.personio.synthetics.e2e.E2EBrowserTest.*")
```

Comment:
Are we planning to remove this exclusion with the future iterations of refactoring?